config/locales/en.yml

```diff
@@ -1301,11 +1301,11 @@ en:
           typed_dag:
             circular_dependency: "The relationship creates a circle of relationships."
           attributes:
-            to:
-              error_not_found: "work package in `to` position not found or not visible"
+            to_id:
+              error_not_found: "not found or not visible"
               error_readonly: "an existing relation's `to` link is immutable"
-            from:
-              error_not_found: "work package in `from` position not found or not visible"
+            from_id:
+              error_not_found: "not found or not visible"
               error_readonly: "an existing relation's `from` link is immutable"
```

Comment:
There is a '.' missing at the end of the `error_not_found` message.

Changing the `error_readonly` message in the same go might be out of scope but easy to do. 

Otherwise, it will read 

"Related work package an existing relation's `from` link is immutable"

It could e.g. be changed to

"cannot be changed for existing relations."